.circleci/config.yml

```diff
@@ -86,6 +86,15 @@ jobs:
             docker build -t ${IMAGE_NAME}:${CIRCLE_BRANCH} -f docker/Dockerfile .
             if [[ ${CIRCLE_BRANCH} == "dev" ]]; then docker tag ${IMAGE_NAME}:${CIRCLE_BRANCH} ${IMAGE_NAME}:latest ; fi
 
+      - run:
+          name: Install trivy
+          command: |
+            curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sudo sh -s -- -b /usr/local/bin
+  
+      - run:
+          name: Scan the local image with trivy (light)
+          command: trivy --exit-code 1 --severity CRITICAL --no-progress ${IMAGE_NAME}:latest
```

Comment:
Whoops. I take that back. See the `if` statement above where sometimes the tag isn't `latest`, it's sometimes `${CIRCLE_BRANCH}`.

Reply:
Thank you, nice catch!